pkgs/development/python-modules/typing-inspect/default.nix

```diff
@@ -16,16 +18,26 @@ buildPythonPackage rec {
     sha256 = "1dzs9a1pr23dhbvmnvms2jv7l7jk26023g5ysf0zvnq8b791s6wg";
   };
 
+  patches = lib.optional (pythonAtLeast "3.9") [
```

Comment:
```suggestion
  patches = [
```

Reply:
In the meantime the patch is no longer needed as upstream published a new release.